core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -487,6 +488,26 @@ public List<Address> getWorkerAddresses() {
     return mWorkerConfigStore.getLiveNodeAddresses();
   }
 
+  @Override
+  public List<alluxio.wire.MasterInfo> getMasterInfoList() {
+    List<alluxio.wire.MasterInfo> masterInfoList = new ArrayList<>(mMasters.size());
+    for (MasterInfo master : mMasters) {
+      masterInfoList.add(new alluxio.wire.MasterInfo(master.getId(),
```

Comment:
It's more clear to use a copy constructor or `Object.clone()` here.
Also, is shallow copy enough for `master.mAddress`?